library/swift/src/Envoy.swift

```diff
@@ -49,6 +49,7 @@ public final class Envoy: NSObject {
 extension Envoy: Client {
   public func startStream(with request: Request, handler: ResponseHandler) -> StreamEmitter {
     let httpStream = self.engine.startStream(with: handler.underlyingObserver)
+    NSLog("Sending request with headers: \(request.outboundHeaders())")
```

Comment:
Let's remove this

Reply:
sure. I thought this was a pretty good log line to have. Is there a way to define log levels with NSLog?

Reply:
For the future. I will remove this for now.

Reply:
Nope, unfortunately no log levels with `NSLog`. I think this would be good to have in the demos but maybe not the engine for now